melodic/distribution.yaml

```diff
@@ -7337,13 +7337,13 @@ repositories:
   mvsim:
     doc:
       type: git
-      url: https://github.com/ual-arm-ros-pkg/mvsim.git
+      url: https://github.com/MRPT/mvsim.git
```

Comment:
Looks like we can update the source URL below too. Let me know if you'd like to update this PR before merging.

Reply:
Sure! :+1: 
Done now.